packages/wordpress-theme-boilerplate/lib/commands/create.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+const { promisify } = require( 'util' );
+const fs = require( 'fs' ).promises;
+const simpleGit = require( 'simple-git/promise' );
+const { Octokit } = require( '@octokit/rest' );
+const inquirer = require( 'inquirer' );
+const keytar = require( 'keytar' );
+const replace = require( 'replace-in-file' );
+const { pascalCase, camelCase, paramCase, snakeCase } = require( 'change-case' );
+const terminalLink = require( 'terminal-link' );
+const rimraf = promisify( require( 'rimraf' ) );
+const { log, format } = require( '../logger' );
+const { validateSlug, validatePHPNamespace, validateNotEmpty } = require( '../validation' );
+const { sleep, runShellCommand, runStep } = require( '../utils' );
+
+const TEMPLATE_OWNER = 'wearerequired';
+const TEMPLATE_REPO = 'wordpress-theme-boilerplate'; // @TODO
+const WORKING_DIR = process.cwd();
+
+async function create( command ) {
+	if ( ! command.skipIntro ) {
+		log(
+			format.title( '👋  Welcome to the WordPress Theme Boilerplate CLI Tool\n\n' ) +
+				'This tool will guide you through the setup process.\n' +
+				'Before you can start please make sure you have created a ' +
+				terminalLink(
+					'personal access token for GitHub',
+					'https://github.com/settings/tokens'
+				) +
+				" with the 'repo' scope selected.\nAfter the first run the token gets stored in your system's keychain and will be pre-filled on next runs.\n"
+		);
+
+		const { isReady } = await inquirer.prompt( [
+			{
+				type: 'expand',
+				name: 'isReady',
+				default: 2, // default to help in order to avoid clicking straight through
+				choices: [
+					{ key: 'y', name: 'Yes', value: true },
+					{ key: 'n', name: 'No', value: false },
+				],
+				message: 'Are you ready to proceed?',
+			},
+		] );
+
+		if ( ! isReady ) {
+			log( format.error( '\nAborted.' ) );
+			process.exit();
+		}
+
+		log();
+	}
+
+	// Get token from the keychain.
+	const storedGithubToken = await keytar.getPassword( 'wordpress-plugin-boilerplate', 'github' );
+
+	const {
+		githubToken,
+		projectName,
+		projectSlug,
+		projectURL,
+		themeName,
+		themeDescription,
+		themeSlug,
+		phpNamespace,
+		githubSlug,
+		privateRepo,
+	} = await inquirer.prompt( [
+		{
+			type: 'password',
+			mask: '*',
+			name: 'githubToken',
+			default: storedGithubToken,
+			message: 'GitHub API token:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'projectName',
+			default: 'Project Name',
+			message: 'Enter the name of the project:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'projectSlug',
+			default: ( answers ) => paramCase( answers.projectName ),
+			message: 'Enter the slug of the project:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'projectURL',
+			default: 'Project URL',
+			message: 'Enter the url of the project:',
+			validate: validateNotEmpty,
+		},
```

Comment:
Do we really need these? `projectURL` is the theme URL?

Reply:
Yes, we need these. It could be renamed as theme URL or site URL. It is used to replace example.org links.

Reply:
Let's simplify this by using the GitHub repo URL by default [as done in the plugin boilerplate](https://github.com/wearerequired/wordpress-plugin-boilerplate/blob/aa747f0ec4008f11da010303db72eb20c42bac59/plugin-name.php#L4)?

Reply:
Good idea! The other place it was being used is the Readme but there we could use the project slug instead.
https://github.com/wearerequired/wordpress-theme-boilerplate/search?q=example.org